cloud_governance/policy/aws/ec2_idle.py

```diff
@@ -143,8 +143,11 @@ def __trigger_mail(self, tags: list, resource_id: str, days: int, ec2_type: str
             special_user_mails = self._literal_eval(self._special_user_mails)
             user, instance_name = self._get_tag_name_from_tags(tags=tags, tag_name='User'), self._get_tag_name_from_tags(tags=tags, tag_name='Name')
             to = user if user not in special_user_mails else special_user_mails[user]
+            email_id = self._get_tag_name_from_tags(tags=tags, tag_name='Email')
             ldap_data = self._ldap.get_user_details(user_name=to)
             cc = [self._account_admin, f'{ldap_data.get("managerId")}@redhat.com']
+            if email_id:
```

Comment:
What the purpose of this  email adding ?

Reply:
if user name is not in LDAP, we need to check Email tag.